Trigger/trigger_api.py

```diff
@@ -1408,18 +1408,42 @@ def set_effect(self, trigger_ids: List[int], visible: bool=False, start_delay: i
         """
         self.ctx.SetEffect(Array[Int32](trigger_ids), visible, start_delay, interval)
 
-    def set_event_ui(self, type: int, arg2: str='', arg3: str='', arg4: str='') -> None:
-        """이벤트UI를설정한다
+    def set_event_ui_countdown(self, round_countdown: List[int], script: str='', box_ids: List[str]=[]) -> None:
+        """이벤트UI를설정한다: arg1=2
```

Comment:
Are you able to update the generator to use "type=2" here rather than "arg1=2"?

Reply:
I'm not sure if I can because arg1=0 and 2 don't really use 'type' parameter in the function. arg1 is being used at the functionsplitter. So I'm not really sure if I can rename the arg before using it as the splitter

Reply:
it might work, i guess im not sure either.

Reply:
ya I don't think I can. if I try to rename it, it adds it as a parameter